client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -37,7 +37,7 @@ class CancelPurchaseForm extends React.Component {
 		defaultContent: PropTypes.node.isRequired,
 		onInputChange: PropTypes.func.isRequired,
 		purchase: PropTypes.object.isRequired,
-		selectedSite: PropTypes.object.isRequired,
+		selectedSite: PropTypes.shape( { slug: PropTypes.string.isRequired } ),
```

Comment:
It looks like we only need the selected site slug for this component to work; the rest of the site object is unnecessary. While you're here, would you like to clean that up?

Reply:
I agree with your suggestion, but making the change right now requires changes in places I'd rather avoid for now, so I'm going to hold off.